prowler/providers/aws/aws_provider.py

```diff
@@ -1313,21 +1313,30 @@ def read_aws_regions_file() -> dict:
     return data
 
 
-def get_aws_available_regions() -> set:
+def get_aws_available_regions(partition: str = None) -> set:
```

Comment:
What if we include it under the `AwsProvider`?
```suggestion
def AwsProvider.get_regions_by_partition(partition: str = None) -> set:
```

Reply:
Done